i18n/es/docusaurus-plugin-content-docs/version-3/topics/advanced.md

```diff
@@ -70,8 +70,8 @@ import (
     "log"
     "os"
 
-    "helm.sh/helm/v3/pkg/action"
-    "helm.sh/helm/v3/pkg/cli"
+    "helm.sh/helm/v4/pkg/action"
+    "helm.sh/helm/v4/pkg/cli"
```

Comment:
we don't want to change v3 -> v4 for any of these translation pages under `version-3`, right?